docs/pages/en-us/Architecture.md

```diff
@@ -5,78 +5,224 @@ title: Architecture
-
-![CAF Audit Account 1 Kafka Reject Table Columns](images/account_1RejectTable.png)
-
-The figure shows the `account_1` schema with a `kafka_rej` table and columns for rejected audit event data.
+### Elasticsearch Indices and Type Mappings
+
+On a tenant's first call of the Audit library, an index is created for the them. A tenant's Elasticsearch index, type and document's ID (`index/type/_id`) is seen as `audit_tenant_<tenantId>/cafAuditEvent/<applicationAuditEvent>`.
+
+    GET /audit_tenant_1/_mapping/cafAuditEvent
+    {
+      "audit_tenant_1": {
+        "mappings": {
+          "cafAuditEvent": {
+            "dynamic_templates": [
+              {
+                "CAFAuditString": {
+                  "match": "*_CAStr",
+                  "mapping": {
+                    "type": "keyword"
+                  }
+                }
+              },
+              {
+                "CAFAuditKeyword": {
+                  "match": "*_CAKyw",
+                  "mapping": {
+                    "type": "keyword"
+                  }
+                }
+              },
+              {
+                "CAFAuditText": {
+                  "match": "*_CATxt",
+                  "mapping": {
+                    "type": "text"
+                  }
+                }
+              },
+              {
+                "CAFAuditLong": {
+                  "match": "*_CALng",
+                  "mapping": {
+                    "type": "long"
+                  }
+                }
+              },
+              {
+                "CAFAuditInteger": {
+                  "match": "*_CAInt",
+                  "mapping": {
+                    "type": "integer"
+                  }
+                }
+              },
+              {
+                "CAFAuditShort": {
+                  "match": "*_CASrt",
+                  "mapping": {
+                    "type": "short"
+                  }
+                }
+              },
+              {
+                "CAFAuditDouble": {
+                  "match": "*_CADbl",
+                  "mapping": {
+                    "type": "double"
+                  }
+                }
+              },
+              {
+                "CAFAuditFloat": {
+                  "match": "*_CAFlt",
+                  "mapping": {
+                    "type": "float"
+                  }
+                }
+              },
+              {
+                "CAFAuditDate": {
+                  "match": "*_CADte",
+                  "mapping": {
+                    "type": "date"
+                  }
+                }
+              },
+              {
+                "CAFAuditBoolean": {
+                  "match": "*_CABln",
+                  "mapping": {
+                    "type": "boolean"
+                  }
+                }
+              }
+            ],
+            "properties": {
```

Comment:
Not sure why, but I would have preferred the fixed fields list first in this mapping with the custom ones after.